src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -11,16 +11,21 @@
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(UseContractReferences)' == 'true'" >
```

Comment:
It was UseProjectReferences which is true by default. I've updated it to UseContractReferences so now by default if we don't specify any environment variable it is false and will have the same behavior as it is right now.